src/omnicore/rules.cpp

```diff
@@ -301,6 +308,21 @@ bool IsAllowedOutputType(int whichType, int nBlock)
 }
 
 /**
+ * Adds a pending activation to the PendingActivations vector.
```

Comment:
Originally I was going to have a function to check pending alerts, see if one went active in the block we just processed, and perform any necessary actions.

Then while working it I found the only necessary action was the "X has gone live" notification, which is easily handled via the expiry of the "X is about to go live" notification without using the pending stuff.

So long story short, not sure if I'll use it.  Pending activations at runtime can be seen via the ```omni_getinfo``` RPC call as there will be a alert for it.